docs/reference/kubectl/docker-cli-to-kubectl.md

```diff
@@ -26,44 +26,43 @@ CONTAINER ID        IMAGE               COMMAND                  CREATED
+On a Kubernetes cluster version 1.2, use `--generator=run/v1` to create replication controllers. See [`kubectl run`](/docs/user-guide/kubectl/{{page.version}}/#run) for more information.
+**Note:** `kubectl` commands print the type and name of the resource created or mutated, which can then be used in subsequent commands. You can expose a new Service after a Deployment is created.
 
 ```shell
 # expose a port through with a service
 $ kubectl expose deployment nginx-app --port=80 --name=nginx-http
 service "nginx-http" exposed
 ```
 
-With kubectl, we create a [Deployment](/docs/concepts/workloads/controllers/deployment/) which will make sure that N pods are running nginx (where N is the number of replicas stated in the spec, which defaults to 1). We also create a [service](/docs/user-guide/services) with a selector that matches the Deployment's selector. See the [Quick start](/docs/user-guide/quick-start) for more information.
+By using kubectl, you can create a [Deployment](/docs/concepts/workloads/controllers/deployment/) to ensure sure that N pods are running nginx, where N is the number of replicas stated in the spec and defaults to 1. You can also create a [service](/docs/user-guide/services) with a selector that matches the selector of the Deployment. For more information, see the [Quick start](/docs/user-guide/quick-start).
 
-By default images are run in the background, similar to `docker run -d ...`, if you want to run things in the foreground, use:
+By default images run in the background, similar to `docker run -d ...`, to run things in the foreground, use:
 
 ```shell
 kubectl run [-i] [--tty] --attach <name> --image=<image>
 ```
 
-Unlike `docker run ...`, if `--attach` is specified, we attach to `stdin`, `stdout` and `stderr`, there is no ability to control which streams are attached (`docker -a ...`).
-To detach from the container, you can type the escape sequence which is Ctrl+P
-followed by Ctrl+Q.
+Unlike `docker run ...`, if you specify `--attach`, then you have attach to `stdin`, `stdout` and `stderr`. You cannot control which streams are attached (`docker -a ...`).
+To detach from the container, you can type the escape sequence Ctrl+P followed by Ctrl+Q.
 
-Because we start a Deployment for your container, it will be restarted if you terminate the attached process (e.g. `ctrl-c`), this is different from `docker run -it`.
-To destroy the Deployment (and its pods) you need to run `kubectl delete deployment <name>`.
+Because kubectl run command starts a Deployment for the container, the Deployment restarts if you terminate the attached process by using `ctrl-c`. Thus different from `docker run -it`.
+To destroy the Deployment and its pods you need to run `kubectl delete deployment <name>`.
 
 #### docker ps
 
-How do I list what is currently running? Checkout [kubectl get](/docs/user-guide/kubectl/{{page.version}}/#get).
+To list what is currently running, see [kubectl get](/docs/user-guide/kubectl/{{page.version}}/#get).
```

Comment:
Revise this link (and other similar links in the docs).
`/docs/user-guide/kubectl` is a redirection to `/docs/reference/generated/kubectl/kubectl-options/` now.
We should use the new target for links rather than use the old paths.

Reply:
Incorporated the changes that you suggested. Please review and let me know if you are okay with the changes.